src/FFMpeg/Media/Audio.php

```diff
@@ -84,8 +84,13 @@ public function save(FormatInterface $format, $outputPathfile)
         }
 
         if (null !== $format->getAudioKiloBitrate()) {
-            $commands[] = '-b:a';
-            $commands[] = $format->getAudioKiloBitrate() . 'k';
+            if (!$format->getEnableVbrEncoding()) {
```

Comment:
You also need to consider the case were this parameter will not have been set:

`if (($format->getEnableVbrEncoding() === NULL) || !$format->getEnableVbrEncoding()) {`